src/crypto/tests/CHIPCryptoPALTest.cpp

```diff
@@ -333,33 +360,6 @@ static void TestAES_CCM_128DecryptTestVectors(nlTestSuite * inSuite, void * inCo
     NL_TEST_ASSERT(inSuite, numOfTestsRan > 0);
 }
 
-static void TestAES_CCM_128EncryptNilKey(nlTestSuite * inSuite, void * inContext)
```

Comment:
Any reason  to delete this test?

Reply:
Yes, the test only tested the 'key is null' senario which is useless if we enforce passing the key as a reference.